### ISSUE TITLE
querry on hyperparameters

### ISSUE THREAD

**Issue by Zuhashaik** [LIN-SHANG/InstructERC#2]
How did you find the optimal hyperparameters for training this model?

**Comment by LIN-SHANG** [comment]
As show in my paper, we don not search for the optimal hyperparameter, but select several hyperparameters repeat the experiment to complete the significance test.

**Comment by Zuhashaik** [comment]
okay, but how did you select them? any studies made by seeing the dataset or something, even i'm working on the similar task as this, so this information might help..any if i need any help or suggetions how can i contact you? mail or linkedin or something?

**Comment by LIN-SHANG** [comment]
Regarding the selection of hyperparameters, this question itself is a very hardcore and broad question. You can look at papers from conferences like ICLR, ICML, etc. for in-depth research, such as the work on Hyperparameter Selection for Imitation Learning. But as a user, in the daily tuning process, I think there are two good experiences: 1. Try to keep consistent with mainstream work, look at some networks that are close to the task, and start with it for range search in hyperparameter selection. 2. Flexibly use some hyperparameter tuning tools, such as GridSearchCV and RandomizedSearchCV in Scikit-learn, Hyperopt, Optuna.

In summary, how to choose good hyperparameters is essentially solving a high-dimensional space optimization problem. If you don't have a strong foundation in mathematics and optimization, trying several sets of hyperparameters may be a suitable choice for you.
